cupyx/scipy/sparse/linalg/solve.py

```diff
@@ -1,9 +1,9 @@
-import numpy
+import numpy as np
```

Comment:
CuPy does not accept abbreviated library names, see [this section](https://docs-cupy.chainer.org/en/latest/contribution.html#coding-guidelines):
> we restrict the usage of shortcut symbols in our code base.

Reply:
Another CuPy style guide: https://github.com/cupy/cupy/wiki/Development-policies